CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  TimeFrameSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 04/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct TimeFrameSection: View {
+    
+    @State var selected: TimeFrameItem = .init(timeFrame: .oneDay)
+    
+    private let timeFrameList: [TimeFrameItem] = [
+        .init(timeFrame: .oneDay),
+        .init(timeFrame: .oneWeek),
+        .init(timeFrame: .oneMonth),
+        .init(timeFrame: .oneYear),
+        .init(timeFrame: .fiveYears)
+    ]
+
+    var body: some View {
+        VStack(alignment: .center) {
+            HStack {
+                ForEach(timeFrameList) { item in
+                    Button(item.title) {
+                        // TODO: - handling loading chart logic
+                        selected = TimeFrameItem(timeFrame: item.timeFrame)
+                    }
+                    .frame(width: 46.0, height: 32.0)
+                    .foregroundColor(
+                        selected == TimeFrameItem(timeFrame: item.timeFrame)
+                        ? .white
+                        : Colors.osloGray.swiftUIColor
+                    )
+                    .background(
+                        selected == TimeFrameItem(timeFrame: item.timeFrame)
+                        ? Colors.caribbeanGreen.swiftUIColor
+                        : .clear
+                    )
+                    .cornerRadius(12.0)
+                    .padding(.horizontal, 6.0)
+                }
+            }
+            .padding(.horizontal, 40.0)
+            .padding(.vertical, 24.0)
```

Comment:
Feel free to ignore this 🙏 

If we use the built-in `Picker`, this is the `TimeFrameSection`'s `body`:
```swift
Picker(selection: $selected) {
  ForEach(timeFrameList) { item in
    Text(item.title).tag(item)
  }
} label: {
  Text("")
    .labelsHidden()
  }
  .pickerStyle(.segmented)
```
and this is what it displays:

https://user-images.githubusercontent.com/17972674/210925196-df4be887-841d-4d4b-85d3-c9a196fdd7ed.mp4

We can do a custom one with a name like `SegmentedPicker` or `SegmentedControl`, so it can hide unnecessary implementation detail and can be used in general cases other than for price's dates.
```swift
var body: some View {
  SegmentedPicker(selection: $selected) {  👈  mimics Apple API
    ForEach(timeFrameList) { item in
      Text(item.title)
    }
  }
}
```

I think this is one of chances we should practice write a custom, generic views (exposed a `Binding`). But this can be put off for later 🙏

Reply:
@Thieurom The idea of using a built-in picker is nice, there is just a small concern that at what level we can customize the UI. It can be seen that the default picker display right now looks nowhere near the design and we will need to twist it a little. If it is doable, using the built-in would definitely be the go-to option 🚀

Reply:
Thank you for the suggestion ;) I will give it a try.